lib/std/fmt.zig

```diff
@@ -1181,6 +1181,19 @@ fn bufPrintIntToSlice(buf: []u8, value: anytype, base: u8, uppercase: bool, opti
     return buf[0..formatIntBuf(buf, value, base, uppercase, options)];
 }
 
+pub fn comptimePrint(comptime fmt: []const u8, args: anytype) *const [count(fmt, args)]u8 {
+    comptime var buf: [count(fmt, args)]u8 = undefined;
+    _ = bufPrint(&buf, fmt, args) catch |err| @compileError(err);
+    return &buf;
+}
+
+test "comptimePrint" {
+    comptime {
+        if (!mem.eql(u8, comptimeBufPrint("{}", .{100}), "100"))
+            unreachable;
+    }
```

Comment:
```suggestion
    comptime assert(mem.eql(u8, comptimeBufPrint("{}", .{100}), "100"));
```

Wouldn't this also be possible?

Reply:
Or `testing.expect`. Or `testing.expectEqualSlices`. I'm not sure if that function works at `comptime`, but it guess it should.